crates/biome_grit_formatter/src/context.rs

```diff
@@ -62,6 +62,7 @@ pub struct GritFormatOptions {
     line_width: LineWidth,
     quote_style: QuoteStyle,
     attribute_position: AttributePosition,
+    bracket_spacing: BracketSpacing,
```

Comment:
Has this setting been used? I haven't seen it. Maybe we could add it when needed.